ui/values/localizable/en.go

```diff
@@ -301,6 +301,7 @@ unlockWithPassword = "Unlock with password"
 "setupMixerInfo" = "%v Two dedicated accounts %v mixed %v & %v unmixed %v will be created in order to use the mixer. %v This action cannot be undone.%v"
 "backupInfo" = "%v Please backup your seed words and keep them in a safe place in order to recover your funds if your device gets lost or broken. %v Anyone who has your seed words can spend your funds! Do not share them.%v"
 "signMessageInfo" = "%v Signing a message with an address' private key allows you to prove that you are the owner of a given address to a possible counterparty.%v"
+"securityToolsInfo" = "%v Various tools that help in different aspects of cryptocurrency security will be located here. %v"
```

Comment:
move this to the bottom so we know string that need other language translations.